chromadb/test/ef/test_nomic_ef.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+import pytest
+
+# import requests
+# from requests import HTTPError
+# from requests.exceptions import ConnectionError
+
+from chromadb.utils.embedding_functions import NomicEmbeddingFunction
+
+
+def test_nomic() -> None:
+    """
+    To learn more about the Nomic API: https://docs.nomic.ai/reference/endpoints/nomic-embed-text
+    Export the NOMIC_API_KEY and optionally the NOMIC_MODEL environment variables.
+    """
+    if os.environ.get("NOMIC_API_KEY") is None:
+        pytest.skip("NOMIC_API_KEY environment variable not set. Skipping test.")
+    # try:
+    #     response = requests.get(os.environ.get(???, ""))
+    #     # If the response was successful, no Exception will be raised
+    #     response.raise_for_status()
+    # except (HTTPError, ConnectionError):
+    #     pytest.skip("Nomic API server can't be reached. Skipping test.")
```

Comment:
Nomic doesn't seem to have an easy way to test that the API is up (they have only this single end point currently) so I've messaged them on their Discord to see if they have anything available/that they could add. Otherwise, we should consider some other behavior here for skipping.